frameworks/CSharp/aspnetcore/Benchmarks/Middleware/MultipleQueriesDapperMiddleware.cs

```diff
@@ -37,7 +33,7 @@ public async Task Invoke(HttpContext httpContext)
                 var db = httpContext.RequestServices.GetService<DapperDb>();
                 var rows = await db.LoadMultipleQueriesRows(count);
 
-                var result = JsonConvert.SerializeObject(rows, _jsonSettings);
+                var result = JsonSerializer.Serialize(rows, _serializerOptions);
```

Comment:
Did you consider using the source generator?

Reply:
Not yet in this project. So far only the platform ones which are the most optimized.